.github/workflows/developer_onboarding_notification.yml

```diff
@@ -0,0 +1,83 @@
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
+
+jobs:
+  comment-on-merge:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/merge
+
+      - name: Set intermediate environment variables
+        run: |
+          echo "AUTHOR=${{ github.event.pull_request.user.login }}" >> $GITHUB_ENV
+          echo "REPO=${{ github.repository }}" >> $GITHUB_ENV
+          echo "OWNER=${{ github.repository_owner }}" >> $GITHUB_ENV
```

Comment:
Please use the `env:` yaml syntax for setting these environment variables as shown in GitHub's documentation. Right now, the untrusted variables are being embedded into bash code, so this is vulnerable to a script injection attack

Reply:
Done!